.env.local

```diff
@@ -25,6 +30,7 @@ export SYNCMASTER__AUTH__CLIENT_ID=manually_created
 export SYNCMASTER__AUTH__CLIENT_SECRET=generated_by_keycloak
 export SYNCMASTER__AUTH__REDIRECT_URI=http://localhost:8000/auth/callback
 export SYNCMASTER__AUTH__SCOPE=email
+export SYNCMASTER__AUTH__VERIFY_SSL=False
```

Comment:
Could we rename all Keycloak-related settings to `SYNCMASTER__AUTH__KEYCLOAK__VERIFY_SSL` and so on?

Reply:
Please update .env.docker accordingly